src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -3,6 +3,23 @@
 import org.jabref.logic.l10n.Localization;
 
 public enum KeyBinding {
+    EMACS_DELETE("Emacs delete", Localization.lang("Delete text"), "ctrl+D", KeyBindingCategory.EMACS),
+    EMACS_BACKWARD("Emacs move caret left", Localization.lang("Move caret left"), "ctrl+B", KeyBindingCategory.EMACS),
+    EMACS_FORWARD("Emacs move caret right", Localization.lang("Move caret right"), "ctrl+F", KeyBindingCategory.EMACS),
+    EMACS_WORD_BACKWARD("Emacs move caret to previous word", Localization.lang("Move caret to previous word"), "alt+B", KeyBindingCategory.EMACS),
+    EMACS_WORD_FORWARD("Emacs move caret to next word", Localization.lang("Move caret to next word"), "alt+B", KeyBindingCategory.EMACS),
+    EMACS_BEGINNING("Emacs move caret to beginning of line", Localization.lang("Move caret to beginning of line"), "ctrl+A", KeyBindingCategory.EMACS),
```

Comment:
I thought about this again, and still think it would make sense to add these keybindings as normal keybindings, setting them to "normal" defaults: e.g. `WORD_FORWARD` to `Ctrl + right`, others like `CAPITALIZE` to empty. Then people can customize to whatever they want, e.g. emacs or vi or ...

Reply:
Yes, I started working myself slowly through all the comments. I think I agree with you, a generic keybinding solution really does make sense.